episodes/13-narrative.md

```diff
@@ -67,7 +67,9 @@ Lessons included in the [Carpentries Incubator][carpentries-incubator] will need
 Even with a CC0 license, you will still want to follow best practice in 
 giving attribution to the data provider or collecting agency.
 
-:::::::::::::::: More CC0 License Reading
+:::::::::::::::: callout
```

Comment:
Nice! Does this mean that the div validator helped you out?

Reply:
Since I was getting that [other fenced div issue](https://github.com/carpentries/pegboard/issues/83), I actually found it myself looking for what might have caused that issue.